core/commands/files/files.go

```diff
@@ -235,14 +235,33 @@ Examples:
 			return
 		}
 
+		long, _, _ := req.Option("l").Bool()
```

Comment:
should this be a proper argument? makes it appear in --help and also makes it easier to find the options this accepts, when reading the code